setup.py

```diff
@@ -30,13 +30,20 @@
     "neovi": ["filelock", "python-ics>=2.12"],
     "canalystii": ["canalystii>=0.1.0"],
     "cantact": ["cantact>=0.0.7"],
+    "cvector": ["python-can-cvector"],
     "gs_usb": ["gs_usb>=0.2.1"],
     "nixnet": ["nixnet>=0.3.1"],
     "pcan": ["uptime~=3.0.1"],
+    "remote": ["python-can-remote"],
+    "sontheim": ["python-can-sontheim>=0.1.2"],
     "viewer": [
         'windows-curses;platform_system=="Windows" and platform_python_implementation=="CPython"'
     ],
 }
+# Add a tag that allows the user to install all optional dependencies at once
```

Comment:
I'm not sure about this. Once we migrate to pyproject.toml configuration this might be problematic. I'm not sure if there is syntax like this in PEP 621. This is also out of scope for this PR.

@hardbyte what do you think?

Reply:
Agree, out of scope for this PR.

Reply:
Extra tag removed :thumbsup: